test/e2e/far_e2e_test.go

```diff
@@ -238,6 +201,57 @@ func buildNodeParameters(clusterPlatformType configv1.PlatformType) (map[v1alpha
 	return testNodeParam, nil
 }
 
+// randomizeWorkerNode returns a worker node name which is different than the previous one
+// (on the first call it will allways return new node)
+func randomizeWorkerNode(nodes *corev1.NodeList) string {
+	nodeName := previousNodeName
+	for previousNodeName == nodeName {
+		// Generate a random seed based on the current time
+		rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
when you create you own rand, why don't you use it? ;)

Reply:
The idea is that in the test setup there are three worker nodes, and the rand function uses a seed variable (in this case current time in nano seconds) to randomly pick a number. I use `previousNodeName` to verify on the second test that the selected worker node is different than the previous one. 
Since there are only 3 worker nodes I want to avoid choosing the previous node, then I have added this logic so we won't do the E2E test for only one node.

Reply:
I understand what you want to do. The point is: `rand.New()` returns a new rand. But you don't use it. You use the default rand.

Reply:
oops, now it used that 😆